colors.c

```diff
@@ -0,0 +1,29 @@
+#include<stdio.h>
+#include<conio.h>
+void main()
+{
+    int r,g,b;   /*Colour input of RCB type*/
+    float c,m,y,k,w=0.0;
+    printf("Enter the value for red, blue and green"); 
+    scanf("%d %d %d",&r,&b,&g);
+    if (r/255 > g/255 && r/255 > b/255)
```

Comment:
It requires some prior color combination knowledge to understand this program. Could you please document it - may be by commenting or by having a separate .md file - so that its readablity can get improved?